app/android/src/uk/co/lutraconsulting/PositionTrackingService.java

```diff
@@ -143,22 +143,22 @@ public int onStartCommand( Intent intent, int flags, int startId ) {
         NotificationChannel serviceChannel = new NotificationChannel(
             CHANNEL_ID,
             "Foreground Service Channel",
-            NotificationManager.IMPORTANCE_HIGH
+            NotificationManager.IMPORTANCE_LOW
         );
         
         NotificationManager manager = getSystemService( NotificationManager.class );
         manager.createNotificationChannel( serviceChannel );
 
         // Build notification for position tracking
-        Intent notificationIntent = new Intent( this, PositionTrackingService.class );
+        Intent notificationIntent = new Intent( this, InputActivity.class );
```

Comment:
Is this the fix for opening the app when clicked on notification? :)

Reply:
Yes :slightly_smiling_face: